static/src/javascripts/bootstraps/standard/atoms.js

```diff
@@ -26,6 +26,42 @@ const bootstrapAtom = <A>(atomMaker: AtomMaker<A>, atomType: AtomType) => {
     });
 };
 
+const initCharts = () => {
+    const iframes: HTMLIFrameElement[] = ([
+        ...document.querySelectorAll('.atom--chart > .atom__iframe'),
+    ]: any);
+
+    window.addEventListener('message', event => {
+        const iframe: ?HTMLIFrameElement = iframes.find(i => {
+            try {
+                return i.name === event.source.name;
+            } catch (e) {
+                return false;
+            }
+        });
+        if (iframe) {
+            try {
+                const message = JSON.parse(event.data);
+                switch (message.type) {
+                    case 'set-height':
+                        iframe.height = message.value;
+                        break;
+                    default:
+                }
+                // eslint-disable-next-line no-empty
+            } catch (e) {}
+        }
+    });
+
+    iframes.forEach(iframe => {
+        const src = (iframe.getAttribute('srcdoc') || '')
+            .replace(/<gu-script>/g, '<script>')
+            // eslint-disable-next-line no-useless-concat
+            .replace(/<\/gu-script>/g, '<' + '/script>');
```

Comment:
`'<' + '/script>'` why are we doing this? Looks interesting!

Reply:
Browsers have a security mechanism to prevent script injection, that's the only solution I know of to circumvent it.